docs/source/gpus/multigpu-partition-access.md

```diff
@@ -0,0 +1,94 @@
+# Access to Multi-GPU Partition
+The `multigpu` partition in the HPC cluster allows users for extensive parallel processing. This setup is ideal for applications that require significant computational power, such as deep learning, scientific simulations, and large-scale data analysis.
+
+## Applying for Multi GPU reservation
+
+### Step 1
+
+- Use the partition application form link in here: [HPC Partition Request](https://bit.ly/NURC-PartitionAccess)
+- Select under "Partition Type" - `Multigpu - Partition (Testing Access)`
+- Fill in the number of GPUs and the type of GPU you'd like to test on. Please also provide a short description of your expected testing workload.
+- Submit the form.
+- RC will then contact you with further details for accessing a multi-gpu node. RC team will need an estimate of the duration needed for testing (preferably less than 24 hrs.)
+
+### Step 2
+
+- To check your reservation
+    
+    ```bash
+    scontrol show reservation=<reservation_name>
+    
+    #<reservation_name> will be in the details provided by the RC team.
+    ```
+    
+- To run an {term}`Interactive Job`, for example, using V100-sxm2 with 4 GPUs, use the command below
+
+```bash
+srun -p reservation --reservation=<reservation_name> --gres=gpu:v100-sxm2:4 --time=24:00:00 -N 1 --pty /bin/bash
+```
+
+***Note: Your local machine should be active for successful execution of the Job***
+
+- **To run {term}`Non-interactive job`, you can use the following command**
+    
+    1- Create a script
+    
+    ```bash
+    nano my_job.sh
```

Comment:
I might make a comment here there you can use the text editor of your choice. Just so no one thinks they have to use nano.